packages/default/scss/panelbar/_layout.scss

```diff
@@ -75,25 +81,73 @@
                 cursor: default;
                 transition: $transition;
             }
+
+            // Hierarchy items
+            &.k-level-2 {
+                .k-link {
+                    padding-left: $panelbar-item-padding-x * 2;
+                }
+            }
+            &.k-level-3 {
+                .k-link {
+                    padding-left: $panelbar-item-padding-x * 3;
+                }
+            }
+            &.k-level-4 {
+                .k-link {
+                    padding-left: $panelbar-item-padding-x * 4;
+                }
+            }
```

Comment:
How about using a variable that can be changed and a cycle, so developers that need more levels can easily change this and build the theme?